timescale-forge/vpc-peering.md

```diff
@@ -0,0 +1,269 @@
+
+<highlight type="tip">
+If you need to, you can use another security group which already exists in your VPC,
+however, for simplicity we will assume the creation of a new security group.
+</highlight>
+
+<img class="main-content__illustration" src="aws-create-security-group.png" alt="The AWS Security Groups dashboard"/>
+
+From the `Create security group` view, enter a name for your security group. Add whatever
+content you would like to the description field. For the `VPC` field, select the VPC
+which has been peered with your Forge VPC.
+
+No inbound rules are required, so leave the inbound rules section empty.
+
+In the outbound rules section, select `Custom TCP` for the rule type. The protocol
+should remain as TCP. The port range should be `5432`, which is the port which will
+be used to connect to your Timescale Forge services. The Destination should be set
+to `Custom` and the value should be the CIDR block of your Forge VPC.
+
+<highlight type="tip">
+AWS may pre-populate the `Destination` column with the value `0.0.0.0/0`. Though this
+value will certainly work, it is more "open" than needed, and should be deleted.
+</highlight>
+
+Finally, click `Create security group`. With this step, you will now be able to
+connect to any of your Timescale Forge services attached to your peered VPC. In the next
+section, you will learn how to create a Timescale Forge service with a VPC attachment.
+
+## Create a service with VPC attachment
+In the Timescale Forge console, navigate to the
+[Services Dashboard](https://console.forge.timescale.com/dashboard/services). Click
+`Create service` and select the compute and disk size which fits your needs.
+
+Expand the dropdown menu under the `Select a VPC` step and select the VPC you created
+previously. If you have multiple VPCs, select the VPC which you want your new service
+to be attached to.
+
+Click `Create Service`, and Timescale Forge will create your new service. Due to
+selecting a VPC during setup, your new service will be created with an attachment to
+your selected VPC.
+
+<img class="main-content__illustration" src="create-service-with-vpc.png" alt="Create new service with VPC attachment"/>
+
+## Migrating a service
+Timescale Forge services may be migrated between VPCs within a Forge project, and may also
+be migrated to and from the public network. Typically, once you have attached your service
+to a VPC, it should remain attached to ensure that your applications running in your AWS
+VPC will have continued connectivity to your service.
+
+<highlight type="warning">
+Timescale Forge uses a different DNS name for a Timescale service once it has been attached
+to a VPC. This means that you will need to update your connection string if migrating a service
+from the public internet into a VPC, or vice-versa.
+</highlight>
+
+To migrate a Timescale Forge service between VPCs, or to migrate to/from the public network,
+navigate to the operations page of your service. First, navigating to the
```

Comment:
Maybe replace `navigating` with `go` so that we don't keep repeating the same word (feels clunky). Can also remove `First` since we never follow up with a `second` or `third`

Reply:
Yea, I conjugated incorrectly. I already fixed that based on my previous self-review. I actually changed it to chop out some of the intermediate content, and it now reads:

> To migrate a Timescale Forge service between VPCs, or to migrate to/from the public network,
> navigate to the [Services Dashboard](https://console.forge.timescale.com/dashboard/services),
> select the service you wish to migrate, click the `Operations` tab in the service details view,
> and click the `VPC` tab within the `Operations` view. From this view, you have a few options
> depending on the state of your service.

Reply:
Thoughts?

Reply:
much better in the re-write 👍  (As an aside, I'll be writing some guidance on verbs in the next week or so, as well).